migration_scripts/upgrade_from_2.4_to_2.5.py

```diff
@@ -0,0 +1,30 @@
+"""
+Simply makes UsefulLife an int.
+"""
+import argparse
+from lxml import etree as ET
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("file_path", type=str, help="path to file to update")
+    return parser.parse_args()
+
+def update_usefulLife(tree):
+    for usefulLife in tree.findall('.//*{http://buildingsync.net/schemas/bedes-auc/2019}UsefulLife'):
+        usefulLife.text = usefulLife.text.split(".", 1)[0]
```

Comment:
```suggestion
        usefulLife.text = str(int(usefulLife.text))
```
Currently, we only round down, is this right? If so, would this also round up when necessary?